learningresources/models.py

```diff
@@ -228,3 +202,46 @@ class Meta:  # pylint: disable=missing-docstring
             RepoPermission.add_edit_metadata,
             RepoPermission.manage_repo_users,
         )
+
+
+def get_preview_url(resource, org=None, course_number=None, run=None):
```

Comment:
if this has become an API, why is it here in `models.py` and not in `api.py`? Circular imports?